src/DI/CacheConsoleExtension.php

```diff
@@ -33,12 +32,12 @@ public function loadConfiguration(): void
 		$config = $this->config;
 
 		// Default values cannot be in schema, arrays are merged by keys
-		if ($config->purge === []) {
-			$config->purge = Helpers::expand(['%tempDir%/cache'], $builder->parameters);
+		if (!isset($config['purge']) || ($config['purge'] === [])) {
+			$config['purge'] = Helpers::expand(['%tempDir%/cache'], $builder->parameters);
```

Comment:
Why did you change stdClass to array?

Reply:
I have changed `stdClass` to `array` because the default configuration (when extensions are only registered) is [an empty array](https://github.com/nette/di/blob/master/src/DI/CompilerExtension.php#L29). Another option is explicitly use `stdClass` as default value in `ConsoleBridgesExtension::getConfigSchema`.